Add `Project Loved: December 2022` news post (#8458)
ppy/osu-wiki: news/2022-12-15-project-loved-december-2022.md

---
layout: post
title: "Project Loved: December 2022"
date: 2022-12-15 18:00:00 +0000
---

> Celebrate Christmas with some festive Loved gifts! 🎄🎁

Reviewer: ``` Suggestion
Celebrate the holidays with some festive Loved gifts! 🎄🎁
```

- [Results for osu!](https://osu.ppy.sh/community/forums/posts/1675562)
- [Results for osu!taiko](https://osu.ppy.sh/community/forums/posts/1675561)
- [Results for osu!catch](https://osu.ppy.sh/community/forums/posts/1675560)
- [Results for osu!mania](https://osu.ppy.sh/community/forums/posts/1675559)

Before we move on with this month's picks, let's talk about some changes within the Project Loved team:

- We'd like to bid farewell to [Noffy](https://osu.ppy.sh/users/1541323), [Nokashi](https://osu.ppy.sh/users/5431196), and [Zeus-](https://osu.ppy.sh/users/5464437), as they've resigned from the team very recently. Thank you for making Loved awesome by making sure that all picks have been thoroughly checked and prepared for the Loved category in terms of integral metadata and moderation checks!
- On the other hand, say hello to [Maxus](https://osu.ppy.sh/users/4335785)! He will be joining the metadata team to make sure that picks are in pristine condition before receiving their well-earned leaderboards.

> - A new drummer approaches! [Eriha](https://osu.ppy.sh/users/16320311) joins the osu!taiko Loved captains in order to help with bringing some of the community's most adored beatmaps to justice by granting them all a chance in the Loved polls.

Reviewer 2: ```suggestion
- A new drummer approaches! [Eriha](https://osu.ppy.sh/users/16320311) joins the osu!taiko Loved captains in order to help bring some of the community's most adored beatmaps to justice by granting them all a chance in the Loved polls.
```

> That should be all in terms of updates for this months. Don't forget if you're interested in participating in Project Loved discussions of a certain game mode, or have any questions, that you can reach out in one of the Project Loved Discord servers listed below:

Reviewer 2: ```suggestion
That should be all in terms of updates for this month. Don't forget if you're interested in participating in Project Loved discussions of a certain game mode, or have any questions, that you can reach out in one of the Project Loved Discord servers listed below:
```

[![osu!catch Project Loved Discord server banner](/wiki/shared/news/banners/project-loved/banner-catch.png)](https://discord.gg/phgtyS4UCh)

[![osu!mania Project Loved Discord server banner](/wiki/shared/news/banners/project-loved/banner-mania.png)](https://discord.gg/Ededv7m)

### Navigation

- [osu!](#osu!)
- [osu!taiko](#osu!taiko)
- [osu!catch](#osu!catch)
- [osu!mania](#osu!mania)

## osu!

The osu! Loved candidates were chosen by [DeviousPanda](https://osu.ppy.sh/users/4966334), [Mismagius](https://osu.ppy.sh/users/19048), [rHO](https://osu.ppy.sh/users/1629553), [Skill](https://osu.ppy.sh/users/7802027) and [Wixonater](https://osu.ppy.sh/users/10083084)!

---

[![](/wiki/shared/news/2022-12-15-project-loved-december-2022/1360992.jpg)](https://osu.ppy.sh/community/forums/topics/{{TOPIC_ID}})

[wa. - Black Lotus](https://osu.ppy.sh/beatmapsets/1360992#osu) by [blixys](https://osu.ppy.sh/users/13015896)\
200 BPM, 1:47 | 8.37★\
*description written by [rHO](https://osu.ppy.sh/users/1629553)*

If you don't know who [blixys](https://osu.ppy.sh/users/13015896) is, now is the time to meet them and their beautifully chaotic maps.

> Speaking in a language that mostly consists of heavily constructed rhythms, map structure and sliders, blixys shows us what they are capable of — capturing destruction and rebirth in their map *Black Lotus*. Using their mapping and object manipulation skills to the max, blixys successfully pulls off an jaw-dropping beatmap full of flowers and lotuses. I can hear the people saying *"Beatmaps are never that serious and/or poetic, rHO!"* comments already, but if you don't believe me, take a gander and you'll see what I'm talking about 😛

Reviewer: Sorry, not sorry for the oxford comma
``` Suggestion
Speaking in a language that mostly consists of heavily constructed rhythms, map structure, and sliders, blixys shows us what they are capable of — capturing destruction and rebirth in their map *Black Lotus*. Using their mapping and object manipulation skills to the max, blixys successfully pulls off a jaw-dropping beatmap full of flowers and lotuses. I can hear the people saying *"Beatmaps are never that serious and/or poetic, rHO!"* comments already, but if you don't believe me, take a gander and you'll see what I'm talking about 😛
```

Reviewer 3: news posts have more relaxed writing rules, including the oxford comma being optional

Reviewer 2: still would be good to fix an -> a but yeah

[HachiojiP x Giga feat. Hatsune Miku, Kagamine Rin - GimmexGimme](https://osu.ppy.sh/beatmapsets/1238827#osu) by [Mir](https://osu.ppy.sh/users/8688812)\
105 BPM, 3:36 | 5.63★\
*description written by [DeviousPanda](https://osu.ppy.sh/users/4966334)*

Mir's mapset of the popular vocaloid track *GimmexGimme* is one that makes use of accelerating sliders to truly map the song to its highest potential while still being approachable by the wider player base.

This map is an absolute treat for players of any skill level, falling in the mid-5\* range and with patterns so fun that it feels like the song was made around the map, so be sure to check this out and give it a few tries, and I'll guarantee that it will be a great experience!

[![](/wiki/shared/news/2022-12-15-project-loved-december-2022/322808.jpg)](https://osu.ppy.sh/community/forums/topics/{{TOPIC_ID}})

[LIQU@. - Yotogibanashi no Kamikakushi](https://osu.ppy.sh/beatmapsets/322808#osu) by [Ekoro](https://osu.ppy.sh/users/284905)\
240 BPM, 1:48 | 8.40★\
*description written by [Skill](https://osu.ppy.sh/users/7802027)*

Ever thought about a Time Freeze II..? Yes? Well, it's here in December's voting.

> \[Ekoro](https://osu.ppy.sh/users/284905)'s take on LIQU@.'s *Yotogibanashi no Kamikakushi* is very reminiscent of his own very popular map [UNDEAD CORPORATION - Everything will Freeze [Time Freeze\]](https://osu.ppy.sh/beatmapsets/158023#osu/555797). Just like Time Freeze, this map features 240 BPM death/spaced streams, along with the iconic squares littered throughout the map. The main difference is that this song only sits at 1:48 in length. Plenty of room for some retrying, and world class score potential!

Reviewer 2: ```suggestion
[Ekoro](https://osu.ppy.sh/users/284905)'s take on LIQU@.'s *Yotogibanashi no Kamikakushi* is very reminiscent of his own very popular map [UNDEAD CORPORATION - Everything will Freeze [Time Freeze\]](https://osu.ppy.sh/beatmapsets/158023#osu/555797). Just like Time Freeze, this map features 240 BPM death/spaced streams, along with the iconic squares littered throughout the map. The main difference is that this song only sits at 1:48 in length. Plenty of room for some retrying, and world class score potential!
```

Reviewer 2: Seems the backslash broke the link

Reviewer 2: missing backslash on diff name which broke the embed

> \[Ekoro](https://osu.ppy.sh/users/284905)'s take on LIQU@.'s *Yotogibanashi no Kamikakushi* is very reminiscent of his own very popular map [UNDEAD CORPORATION - Everything will Freeze [Time Freeze\]](https://osu.ppy.sh/beatmapsets/158023#osu/555797). Just like Time Freeze, this map features 240 BPM death/spaced streams, along with the iconic squares littered throughout the map. The main difference is that this song only sits at 1:48 in length. Plenty of room for some retrying, and world class score potential!

Reviewer 2: ```suggestion
\[Ekoro](https://osu.ppy.sh/users/284905)'s take on LIQU@.'s *Yotogibanashi no Kamikakushi* is very reminiscent of his own very popular map [UNDEAD CORPORATION - Everything will Freeze [\Time Freeze\]](https://osu.ppy.sh/beatmapsets/158023#osu/555797). Just like Time Freeze, this map features 240 BPM death/spaced streams, along with the iconic squares littered throughout the map. The main difference is that this song only sits at 1:48 in length. Plenty of room for some retrying, and world class score potential!
```

180 BPM, 3:59 | 7.69★\
*description written by [-NoName-](https://osu.ppy.sh/users/10400730)*

Our very own [Remuring](https://osu.ppy.sh/users/6522146), who we recently welcomed to the team of osu!mania captains last month, has a map featured in this month's Loved voting: *Ano Yume o Nazotte*!

Created in collaboration with [Pengdoll](https://osu.ppy.sh/users/6392061), this has mapping styles you would typically see from both of them. We captains aim to kill two birds with one stone with this nomination, and we feel that this delivers some of the best charting from both creators here. Typical to LN charts like these, you'll see a lot of notes in between columns occupied by LNs, which is great for increasing your finger strength overall.

Aside from the above, we know that many people in the community are fans of YOASOBI's music, so we know those fans will be happy to see another YOASOBI chart in the Loved voting. If you want to see a good LN collaboration practice chart in Loved, or you just want to jam to YOASOBI, remember to give *Ano Yume o Nazotte* your support!

---

> From [everyone working on Project Loved](https://loved.sh/contributors), we hope you enjoyed this month's selection, and are looking forward to the next! The favoured beatmaps will hit the [Loved beatmap listing](https://osu.ppy.sh/beatmapsets?s=loved) shortly after the polls end. So don't forget to stay hydrated and vote on your favourite beatmaps in order for that to happen!

Reviewer 2: ```suggestion
From [everyone working on Project Loved](https://loved.sh/contributors), we hope you enjoyed this month's selection and are looking forward to the next! The favoured beatmaps will hit the [Loved beatmap listing](https://osu.ppy.sh/beatmapsets?s=loved) shortly after the polls end. So don't forget to stay hydrated and vote on your favourite beatmaps in order for that to happen!
```

> More info about Project Loved can be found on its [wiki page](/wiki/Community/Project_Loved) and [website](https://loved.sh/). Stop by the `#osu-loved` channel of the [osu!dev Discord server](https://discord.gg/ppy) if you have any questions, concerns, or are interested in helping out.

Reviewer: ```Suggestion
More info about Project Loved can be found on its [wiki page](/wiki/Community/Project_Loved) and [website](https://loved.sh/). If there are any questions, comments, concerns, or you are interested in helping out, please stop by the `#osu-loved` channel in the [osu!dev Discord server](https://discord.gg/ppy).
```

Author: prefer current wording, we already used it many times too
